librz/asm/arch/arc/gnu/arcompact-dis.c

```diff
@@ -3731,7 +3731,7 @@ int ARCompact_decodeInstr(bfd_vma address, /* Address of this instruction.  */
 	fprintf_ftype func = info->fprintf_func;
 	int bytes;
 	int lowbyte, highbyte;
-	char buf[256];
+	char buf[allOperandsSize + 1];
```

Comment:
some compilers does not like this. use malloc

Reply:
`allOperandsSize` is an enum constant so there should't be a problem

Reply:
oh, then yeah. sorry

Reply:
if this was rizin code, it would be in all caps